### DISCUSSION TITLE
Possible to resize image based on ratio?

### DISCUSSION THREAD

**Discussion by BrianCrowe3** [chaiNNer-org/chaiNNer#2924] [Q&A]
What I want to do is bulk resize photos based on their ratios, i.e. if their ratio is 0.70 or higher it'd resize them by height, if their ratio is 0.69 or lower it'd resize them by width.

I've played around with most of the nodes and it seems _almost_ possible, but just beyond my grasp. What I'm doing is feeding the "Get Dimensions" node into the "Math" node and dividing the width by the height, but I haven't found a way to do anything useful with that result. 

Is there someway to get this working? Ideally there'd be some sort of "If this, then that" Switch box node that changes where the image is sent based on input from a different node, allowing you to feed the image into different "Resize to Side" nodes based on the value provided by the "Math" node.

**Comment by joeyballentine** [comment]
If you want to try out the [nightly version](https://github.com/chaiNNer-org/chaiNNer-nightly/releases), it has support for creating true/false values and using a condition node
